include/dxc/HLSL/DxilSignatureAllocator.h

```diff
@@ -110,7 +110,7 @@ class DxilSignatureAllocator {
     // - for occupied components, they signify element flags
     // - for unoccupied components, they signify conflict flags
     uint8_t Flags[4];
-    DXIL::InterpolationMode Interp : 4;
+    DXIL::InterpolationMode Interp : 8;
```

Comment:
This is actually a good catch by the compiler. The value assigned to `Interp` is a `uint8_t` (DXIL::InterpolationMode)
+1